ntbs-service/Pages/Index.cshtml.cs

```diff
@@ -75,8 +75,34 @@ private async Task SetUserNotificationsAsync()
         {
             var draftNotificationsQueryable = _notificationRepository.GetDraftNotificationsIQueryable();
             var recentNotificationsQueryable = _notificationRepository.GetRecentNotificationsIQueryable();
-            DraftNotifications = (await _authorizationService.FilterNotificationsByUserAsync(User, draftNotificationsQueryable)).Take(10).ToList();
-            RecentNotifications = (await _authorizationService.FilterNotificationsByUserAsync(User, recentNotificationsQueryable)).Take(10).ToList();
+            DraftNotifications =
+                (await _authorizationService.FilterNotificationsByUserAsync(User, draftNotificationsQueryable))
+                .Take(10)
+                .Select(n => new NotificationForHomePage
+                {
+                    NotificationId = n.NotificationId,
+                    CreationDate = n.CreationDate,
+                    NotificationDate = n.NotificationDate,
+                    GivenName = n.PatientDetails.GivenName,
+                    FamilyName = n.PatientDetails.FamilyName,
+                    TbServiceName = n.HospitalDetails.TBService.Name,
+                    CaseManagerName = n.HospitalDetails.CaseManager.DisplayName
+                })
```

Comment:
If you wanted you could commonise this by making it an extension method on an `IQueryable<Notification>`.

It's not the end of the world if you don't though.